packages/vega-selections/src/selectionTest.js

```diff
@@ -22,7 +22,7 @@ function testPoint(datum, entry) {
 
     if (isDate(dval)) dval = toNumber(dval);
     if (isDate(values[i])) values[i] = toNumber(values[i]);
-    if (isDate(values[i][0])) values[i] = values[i].map(toNumber);
+    if (values[i] && isDate(values[i][0])) values[i] = values[i].map(toNumber);
```

Comment:
Thanks for catching this @jonmmease and apologies for the delay. I think it would be more accurate to switch to this because the rest of the statement (particularly `values[i].map`) assumes that `values[i]` is an Array. 

```suggestion
    if (isArray(values[i]) && isDate(values[i][0])) values[i] = values[i].map(toNumber);
```

(`isArray` is already imported from `vega-util` so this should be ok.)

Reply:
Sounds good!